IdentityCore/src/oauth2/account/MSIDAccountIdentifier.m

```diff
@@ -26,8 +26,8 @@
 #import "MSIDMaskedHashableLogParameter.h"
 #import "MSIDMaskedUsernameLogParameter.h"
 
-static NSString *const MSID_ACCOUNT_DISPLAYABLE_ID_JSON_KEY = @"account_displayable_id";
-static NSString *const MSID_ACCOUNT_HOME_ID_JSON_KEY = @"account_home_id";
+static NSString *const MSID_ACCOUNT_DISPLAYABLE_ID_JSON_KEY = @"username";
```

Comment:
was this used anywhere else? making sure we don't break backward compatibility.

Reply:
It wasn't used previously. I introduced these constants for JSON serialization.